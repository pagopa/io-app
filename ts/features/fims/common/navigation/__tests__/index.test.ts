import { FIMS_ROUTES } from "..";

describe("FIMS_ROUTES", () => {
  it("should match expected values", () => {
    expect(FIMS_ROUTES).toMatchSnapshot();
  });
});
