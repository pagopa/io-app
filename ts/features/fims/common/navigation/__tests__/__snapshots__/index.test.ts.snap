// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FIMS_ROUTES should match expected values 1`] = `
{
  "CONSENTS": "FIMS_SSO_CONSENTS",
  "HISTORY": "FIMS_HISTORY",
  "MAIN": "FIMS_MAIN",
}
`;
